workshops/erc20-mts/scripts/wasm-tx-increment.sh

```
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+source .local.env
+
+if [ -z "$WASM_CONTRACT" ]; then
+  echo "⚠️  Error: WASM_CONTRACT env variable is not set"
+  exit 1
+fi
+
+USER_INJ_ADDRESS=$(injectived keys show $USER --keyring-backend=test -a)\
```

Shellcheck (0.10.0)

[warning] USER_INJ_ADDRESS appears unused. Verify use (or export if used externally).

(SC2034)